todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/statistics/ui/controller.kt

```diff
@@ -16,26 +16,22 @@
  * limitations under the License.
  * -/-/-
  */
-package com.example.android.architecture.blueprints.todoapp.statistics;
+package com.example.android.architecture.blueprints.todoapp.statistics.ui
 
-import com.example.android.architecture.blueprints.todoapp.statistics.domain.StatisticsEffect;
-import com.example.android.architecture.blueprints.todoapp.statistics.domain.StatisticsEvent;
-import com.example.android.architecture.blueprints.todoapp.statistics.domain.StatisticsLogic;
-import com.example.android.architecture.blueprints.todoapp.statistics.domain.StatisticsState;
-import com.spotify.mobius.MobiusLoop;
-import com.spotify.mobius.android.MobiusAndroid;
-import com.spotify.mobius.rx2.RxMobius;
-import io.reactivex.ObservableTransformer;
+import com.example.android.architecture.blueprints.todoapp.statistics.StatisticsEffect
+import com.example.android.architecture.blueprints.todoapp.statistics.StatisticsEvent
+import com.example.android.architecture.blueprints.todoapp.statistics.StatisticsState
+import com.example.android.architecture.blueprints.todoapp.statistics.init
+import com.example.android.architecture.blueprints.todoapp.statistics.update
+import com.spotify.mobius.Update
+import com.spotify.mobius.android.MobiusAndroid
+import com.spotify.mobius.rx2.RxMobius
+import io.reactivex.ObservableTransformer
 
-public class StatisticsInjector {
-  public static MobiusLoop.Controller<StatisticsState, StatisticsEvent> createController(
-      ObservableTransformer<StatisticsEffect, StatisticsEvent> effectHandler,
-      StatisticsState defaultState) {
-    return MobiusAndroid.controller(createLoop(effectHandler), defaultState);
-  }
+fun createController(
+        effectHandler: ObservableTransformer<StatisticsEffect, StatisticsEvent>,
+        defaultState: StatisticsState) = MobiusAndroid.controller(createLoop(effectHandler), defaultState)!!
```

Comment:
Why do we need the `!!`?

Reply:
The IDE is complaining that `MobiusAndroid.controller` may return `null`. Compilation passes. I'll remove it.